frontend/client/components/ContributionModal/index.tsx

```diff
@@ -127,17 +128,14 @@ class ContributionModal extends React.Component<Props, State> {
           message="This contribution will not be attributed"
           description={
             <>
-              Your contribution will show up without attribution. Even if you're logged
-              in, the contribution will not appear anywhere on your account after you
-              close this modal.
+              Your contribution will show up without attribution.
               <br /> <br />
               ZF Grants is unable to offer refunds for non-attributed contributions. If
               refunds for this campaign are issued, your contribution will be treated as a
               donation to the Zcash Foundation.
               <br /> <br />
-              If you would like to have your contribution attached to an account and
-              remain eligible for refunds, you can close this modal, make sure you're
-              logged in, and don't check the "Contribute without attribution" checkbox.
+              If you would like your contribution to be eligible for refund, you may close
```

Comment:
This language should be adjusted to say

If you would like your contribution to be eligible for refund, you may close
 
    
 
 
This language should be adjusted to say...

If you would like your contribution to be eligible for refund, ensure that you have set a refund address in your ~account settings~(linked)

Reply:
I'll run through this and see where the language pops up. This is only correct if it's for logged in users.